index.yaml

```diff
@@ -57,6 +57,21 @@ distributions:
       uxv8: lunar/release-xenial-arm64-build.yaml
     source_builds:
       default: lunar/source-build.yaml
+  melodic:
+    doc_builds:
+      default: melodic/doc-build.yaml
+      released-packages-without-doc-job: melodic/doc-released-build.yaml
+    notification_emails:
+    - ros-buildfarm-melodic@googlegroups.com
+    - clalancette+buildfarm@osrfoundation.org
+    release_builds:
+      default: melodic/release-build.yaml
+      ds: melodic/release-stretch-build.yaml
+      dsv8: melodic/release-stretch-arm64-build.yaml
+      uxhf: melodic/release-armhf-build.yaml
+      uxv8: melodic/release-arm64-build.yaml
```

Comment:
These two keys should not be `ux` (for Ubuntu Xenial) but `ub` (for Ubuntu Bionic). The yaml file name included Xenial in Lunar to clarify that the build file only targets a single Ubuntu distro.

Reply:
Ah, thanks.  Fixed in b737bac